pkg/reconciler/trigger/trigger.go

```diff
@@ -236,6 +252,71 @@ func (r *Reconciler) reconcileRetryTopicAndSubscription(ctx context.Context, tri
 	return nil
 }
 
+// getPubsubRetryPolicy gets the eventing retry policy from the Broker delivery
+// spec and translates it to a pubsub retry policy.
+func getPubsubRetryPolicy(spec *eventingduckv1beta1.DeliverySpec) (*pubsub.RetryPolicy, error) {
+	if spec == nil || spec.BackoffDelay == nil || spec.BackoffPolicy == nil {
```

Comment:
Subscriptions should always have a retry policy. If the delivery spec is empty, we should revert to the default retry policy. This will be the case for any brokers that were created before the webhook defaulting was in place.